src/styles/page.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
```

Comment:
It's recommended to create a variable for the font-family to make it easier to maintain and apply consistent typography throughout the project.